romanesco/__main__.py

```diff
@@ -23,6 +23,7 @@ def run(*pargs, **kwargs):
                         url=jobInfo.get('url'), method=jobInfo.get('method'),
                         headers=jobInfo.get('headers')) as jm:
             kwargs['_job_manager'] = jm
```

Comment:
I realize this is older code, but why do we prefix the job manager arg with an underscore? Does it need to be that hidden? Plus it does not match the corresponding status arg that you just now added.